docs/integrations/datadog-rum/installation.md

```diff
@@ -26,7 +26,48 @@ app:
   #   sessionReplaySampleRate: 0
 ```
 
-If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/packages/app/public/index.html#L69) to include the Datadog RUM `init()` section manually.
+If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/e0506af8fc54074a160fb91c83d6cae8172d3bb3/packages/app/public/index.html#L69) to include the Datadog RUM `init()` section manually. If you a Datadog not publishing issue copy and paste this section in to your `packages/app/public/index.html`
```

Comment:
> Datadog not publishing issue

In the text, could you elaborate on this issue? Something like "In case after a proper configuration, the events still aren't being captured, add..."

Reply:
> copy and paste this section in to your `packages/app/public/index.html`

I would recommend specifying where in the `index.html` file the script should be pasted.

Reply:
Thank you very much for these comments. I will work these changes